README.md

```diff
@@ -102,6 +103,12 @@ cause degradation of performance.
 
 Build with zchunk support (Default: ON)
 
+### ``-DWITH_LIBMODULEMD=ON``
+
+Build with libmodulemd support (Default: ON)
+
+Adds support for working with repos containing Fedora Modularity metadata.
```

Comment:
Could you please provide a link here to Fedora Modularity stuff? No one is going to know what that means otherwise...